init.lua

```diff
@@ -6,6 +6,7 @@ VFS.Include('common/numberfunctions.lua')
 VFS.Include('common/stringFunctions.lua')
 VFS.Include('common/tablefunctions.lua')
 Json = Json or VFS.Include('common/luaUtilities/json.lua')
+Spring.Color = Color or VFS.Include('common/luaUtilities/color.lua')
```

Comment:
Shouldn't it be?
```suggestion
Spring.Color = Spring.Color or VFS.Include('common/luaUtilities/color.lua')
```

And perhaps it would be good to put it in `Spring.Utilities.Color`.

Reply:
Moved the module to Spring.Utilities.Color.